{% autoescape off %}\
Denne mail er sendt fra {{ systemname }} til følgende personer:

{% for p in contactpersons %}\
* {{ p.full_name }} <{{ p.email }}>
{% endfor %}\

{% if testsuite.email_message %}\
{{ testsuite.email_message }}
{% else %}\
Der opstod en fejl under afvikling af test-suiten "{{ testsuite.name }}".
{% endif %}\

Afviklingen af testen startede {{ testrun.started |date:'Y-m-d H:i:s' }}\
{%if testrun.ended %}\
 og sluttede {{ testrun.ended |date:'Y-m-d H:i:s' }}\
{% endif %}\
.
{% if failed_measures %}\

Der var problemer med følgende målinger:
{% for measure in failed_measures %}\

Måling: {{ measure }}
Problem: \
{% if measure.failure_reason %}\
{{ measure.failure_reason }}\
{% else %}\
Målingen fejlede eller overskred den angivne tærskelværdi to gange i træk.
{% endif %}
{% endfor %}\
{% else %}\

Fejlen opstod enten uafhængigt af enkelte målinger eller før der blev
foretaget nogen målinger.
{% endif %}\

--
  Applikationsovervågningssystemet
{% endautoescape %}